### PR TITLE
✨ Enable SBOM on Windows

### PR DESCRIPTION
**Pull request by czunker**

### CODE REVIEW

File: sbom/sbom.mql.yaml [hunk 1]
```diff
@@ -21,7 +22,8 @@ packs:
         mql: npm.packages { name version purl cpes.map(uri) files.map(path) }
       - uid: mondoo-sbom-kernel-installed
         filters:
-          - mql: asset.family.contains('linux')
-          - mql: asset.runtime != 'container' && asset.kind != 'container' && asset.kind != 'container-image'
+          - mql: |
```
Comment (chris-rock) [inline sbom/sbom.mql.yaml:25]: why do we need to join this? We can set multiple filters. Is the current approach not working?

File: sbom/sbom.mql.yaml [hunk 2]
```diff
@@ -6,6 +6,7 @@ packs:
     name: Mondoo SBOM
     filters:
       - mql: asset.family.contains("unix")
+      - mql: asset.family.contains('windows')
```
Comment (chris-rock) [inline sbom/sbom.mql.yaml:9]: Multiple entries should be and connected not OR, so this should not work

File: sbom/sbom.mql.yaml (hunk 1 above)
Comment (czunker) [reply sbom/sbom.mql.yaml:25]:
```
filters:
  - mql: ...
  - mql: ...
```
These filters are ORed. On Windows, this resulted in problems with the kernel query.
Windows doesn't match the first filter, but the second.

File: sbom/sbom.mql.yaml (hunk 2 above)
Comment (czunker) [reply sbom/sbom.mql.yaml:9]: Then, we have a bug in the filters. See comment on the kernel query.

File: sbom/sbom.mql.yaml (hunk 1 above)
Comment (chris-rock) [reply sbom/sbom.mql.yaml:25]: My fault, they are or connected

**Review by chris-rock — APPROVED** [review]
Thank you @czunker

File: sbom/sbom.mql.yaml (hunk 1 above)
Comment (czunker) [reply sbom/sbom.mql.yaml:25]: As discussed: https://github.com/mondoohq/cnspec/pull/1304

**Pull request merged**